tests/bootstrap.php

```
@@ -0,0 +1,25 @@
+}
+
+namespace Joomla\CMS\Factory {
+    // not used by sanitizers; declared only so autoload doesn't fail
+}
+
+namespace Joomla\CMS\Uri {
+    // intentionally empty; Uri import was removed from helper.php
+}
+
+// Load the helper (defines ModCbProfileSlimHelper in global namespace).
+require_once __DIR__ . '/../helper.php';
```

PHPStan (2.2.8)

[error] No code may exist outside of namespace {} on line 25


(phpstan.parse)